apps/token/src/hooks/use-get-user-tranche-balances.ts

```diff
@@ -7,20 +7,24 @@ import {
   useAppState,
 } from '../contexts/app-state/app-state-context';
 import { BigNumber } from '../lib/bignumber';
+import { useTranches } from './use-tranches';
 
 export const useGetUserTrancheBalances = (
   address: string,
   vesting: VegaVesting
 ) => {
   const { appDispatch } = useAppState();
+  const { tranches } = useTranches();
```

Comment:
question (non-blocking): seen that the error handling on the UI has been removed in the [other PR](https://github.com/vegaprotocol/token-frontend/pull/1288/files#diff-c71410a400603c063434e1e096a66d6828a875ec0c8a44cd3c7fe5b5bd45c8efL58-L64), and here it's not handled. just curious, is there any particular reason for that?